spec/section_spec.rb

```diff
@@ -41,6 +41,57 @@ class PageWithAnonymousSection < SitePrism::Page
           .with_message(error_message)
       end
     end
+
+    context 'default search arguments' do
+      class PageWithSectionWithDefaultSearchArguments < SitePrism::Page
```

Comment:
I've not fully read through these yet. But this format for tests I prefer a lot! Top work!

Reply:
👍